src/uwtools/config.py

```diff
@@ -476,7 +476,25 @@ def _format_output(self):
         ''' Format the output of the dictionary into a string that
         matches that necessary for a field_table. Return the string'''
 
+        outstring = []
+        for field, settings in self.data.items():
+            outstring.append(f' "TRACER", "atmos_mod", "{field}"')
+            for key, value in settings.items():
+                if isinstance(value, dict):
+                    method_string = f'{" ":7}"{key}", "{value.pop("name")}"'
+                    # all control vars go into one set of quotes
+                    control_vars = [f'{method}={val}' for method, val in value.items()]
+                    # whitespace after the comma matters
+                    outstring.append(f'{method_string}, "{", ".join(control_vars)}"')
+                else:
+                    #formatting of variable spacing dependent on key length
+                    padlength = 13-len(key)
+                    outstring.append(f'{" ":11}"{key}",{" ":{padlength}}"{value}"')
```

Comment:
Please remove the `padlength` part here. It adds accidental complexity. Also, please keep the "indent" the same length as above. 11 seems to be the right one to get all of them to line up under "atmos_mod".

Reply:
Removed and restored, thanks